frontend/dockerfile/dockerfile2llb/convert.go

```diff
@@ -1381,6 +1391,11 @@ func dispatchCopy(d *dispatchState, cfg copyConfig) error {
 
 	var a *llb.FileAction
 
+	dockerIgnoreMatcher, err := patternmatcher.New(cfg.opt.dockerIgnorePatterns)
```

Comment:
Can't this `opt.dockerIgnorePatterns` already be a `dockerIgnoreMatcher` instance?